pkg/protocol/engine/tipmanager/v1/tipmanager.go

```diff
@@ -194,6 +198,17 @@ func (t *TipManager) SelectTips(amount int) (references model.ParentReferences)
 			references[model.StrongParentType] = rootBlocks[:rootBlockCount]
 		}
 
+		selectWeakTips := createUniqueTipSelector(t.weakTipSet)
+		for weakTipCandidates := selectWeakTips(t.optMaxWeakReferences); len(weakTipCandidates) != 0; weakTipCandidates = selectWeakTips(t.optMaxWeakReferences - len(references[model.WeakParentType])) {
+			for _, weakTip := range weakTipCandidates {
+				if tipPool := t.determineTipPool(weakTip, tipmanager.WeakTipPool); tipPool != tipmanager.WeakTipPool {
```

Comment:
This means that if a node doesn't like conflicts of the Weak Tip it will not add a Shallow Like to fix the opinion but rather drop the tip from the weak tip set, correct? So a node only adds weak parents with liked conflicts - do I understand correctly?